nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/impl/TestPropertyCommentManager.java

```diff
@@ -625,13 +645,66 @@ public void testGetCommentAsRegularUser() {
             assertEquals("The user jane does not have access to the comments of document " + doc.getId(),
                     e.getMessage());
         }
+
+        try (CloseableCoreSession janeSession = CoreInstance.openCoreSession(doc.getRepositoryName(), "jane")) {
+            DocumentModelList documents = janeSession.getDocuments(
+                    new DocumentRef[] { new IdRef(comment.getId()), new IdRef(subComment.getId()) });
+            assertEquals(0, documents.size());
+        }
+    }
+
+    @Test
+    public void testQueryCommentAsRegularUser() {
```

Comment:
This test and this [one](https://github.com/nuxeo/nuxeo/pull/3262/files#diff-ce5dd04ad48c6df00884ab89cc9ae552R68) should be pretty the same and should be the abstract test class for the 2nd PR to merge.

Reply:
yes, we were thinking to rework this test class because there is a lot duplicated code (create files, create the same comments...) but initially we  were thinking to add this work on this same PR with perhaps a separate commit